src/main/interface/smartaudio_protocol.c

```diff
@@ -0,0 +1,187 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "platform.h"
+#include "common/crc.h"
+#include "interface/smartaudio_protocol.h"
+
+#define SMARTAUDIO_SYNC_BYTE                0xAA
```

Comment:
`0xaa 0x55` is 'start code', it's a bit strange to split it to sync/header

Reply:
https://gyazo.com/ff97c3aed1a35309d58b5d4e9149749c

Reply:
It calls them a 'start code' when they're together, but each byte is also defined in the spec as 'sync' and 'header' bytes.

Reply:
Or 'start code' (both bytes) acts as sync/header ;-)